examples/node/package.json

```diff
@@ -5,8 +5,9 @@
     "main": "app.js",
     "author": "",
     "license": "Apache 2.0",
+    "type": "module",
     "dependencies": {
         "cli-color": "^1.0.0",
-        "matrix-js-sdk": "^32.0.0"
+        "matrix-js-sdk": "^34.5.0-rc.0"
```

Comment:
Would be nice to install the SDK from `file:../..` instead and then do something on the CI to verify it doesn't regress again. I couldn't think of a good way to do that though.

Reply:
I'm torn on this. On the one hand, I'd like to set an example of how you might actually depend on the js-sdk in a project, which points towards an explicit version requirement here. On the other hand, I'd like to make sure that the examples continue to work as the js-sdk evolves, pointing towards `file:./..`.

Maybe we can use an explicit version, but keep it up-to-date with Renovate?

Reply:
Or actually, I guess we can just add this into https://docs.renovatebot.com/configuration-options/#filematch? Is there a way to test this without merging to `develop`?

Reply:
I have honestly no idea. Renovate is a mystery to me.